tests/test_shutter_widget.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from pymmcore_widgets._shutter_widget import ShuttersWidget
+
+if TYPE_CHECKING:
+    from pymmcore_plus import CMMCorePlus
+    from pytestqt.qtbot import QtBot
+
+
+def test_shutter_widget(qtbot: QtBot, global_mmcore: CMMCorePlus):
```

Comment:
this test is too big.  test smaller things... ie. test:
1.  instantiation on it's own
2. shutter vs multishutter in different tests
3. autoshutter true vs false in different tests
4. each type of global change

... basically, anywhere you'd be tempted to write "# test xyz" ... try to make a new test.  Imagine the test failing, and then trying to figure out _why_ it failed... (smaller tests are easier to get right to the failure))

Reply:
i think this is the only thing this PR is waiting on

Reply:
@tlambert03 should be ok now.